code/modules/surgery/organs/external/spines.dm

```diff
@@ -14,36 +14,27 @@
 
 	bodypart_overlay = /datum/bodypart_overlay/mutant/spines
 
-	///A two-way reference between the tail and the spines because of wagging sprites. Bruh.
-	var/obj/item/organ/external/tail/lizard/paired_tail
-
 /obj/item/organ/external/spines/Insert(mob/living/carbon/receiver, special, movement_flags)
```

Comment:
There are some extreme scenarious where this will break, since not every mob_insertion is an Insert, but I also don't see a better solution with the external organs code we have now (blame me)